packages/mobile/App/services/sync/utils/executeCrud.ts

```diff
@@ -1,131 +0,0 @@
-import { cloneDeep, chunk } from 'lodash';
-import { In, Repository } from 'typeorm';
-
-import { DataToPersist } from '../types';
-import { SQLITE_MAX_PARAMETERS } from '../../../infra/db/limits';
-import { executePreparedInsert } from './executePreparedInsert';
-
-function strippedIsDeleted(row) {
-  const newRow = cloneDeep(row);
-  delete newRow.isDeleted;
-  return newRow;
-}
-
-export const executeInserts = async (
-  repository: Repository<any>,
-  rows: DataToPersist[],
-  insertBatchSize: number,
-  progressCallback: (processedCount: number) => void,
-): Promise<void> => {
-  // can end up with duplicate create records, e.g. if syncAllLabRequests is turned on, an
-  // encounter may turn up twice, once because it is for a marked-for-sync patient, and once more
-  // because it has a lab request attached
-  const deduplicated = [];
-  const idsAdded = new Set();
-  const softDeleted = rows.filter(row => row.isDeleted).map(strippedIsDeleted);
-
-  for (const row of rows) {
-    const { id } = row;
-    if (!idsAdded.has(id)) {
-      deduplicated.push({ ...strippedIsDeleted(row), id });
-      idsAdded.add(id);
-    }
-  }
```

Comment:
@edmofro As part of this change we're thinking of removing this de-duplication logic, as it shouldn't be needed with sync lookup anymore.

Just wanna confirm that we've fully deprecated the old sync system? And we're happy that all deployments going forward are using sync_lookup